app/views/photos/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<div>
+<% @photos.order(created_at: :desc).each do |photo| %>
+  
+    <% if current_user.id == photo.user_id %>
+        <%= image_tag(photo.image.url, alt: 'Image') if photo.image? %>
+    <% end %>
+
+<% end %>
+</div>
+
+<%= link_to new_photo_path do %>
```

Comment:
Be good to style this as a button like the others will add it to the Trello, to be done later.